README.md

```diff
@@ -0,0 +1,54 @@
+# Wording
+
+POI : Point Of Interest = a place on the map with latitude/longitude/description/category/picture
+
+# Database
+
+Database modelization available in the "resources" folder
+
+# Description of our roads:
+
+## Post : 
+
+### Post '/'
```

Comment:
what does look like the data I can send (JSON)

Reply:
done